composer.json

```diff
@@ -5,8 +5,8 @@
     "description": "A pack for the Doctrine ORM",
     "require": {
         "composer/package-versions-deprecated": "*",
-        "doctrine/orm": "^2",
-        "doctrine/doctrine-bundle": "^2",
-        "doctrine/doctrine-migrations-bundle": "^2"
+        "doctrine/orm": "*",
+        "doctrine/doctrine-bundle": "*",
```

Comment:
both these lines must be kept as is: when dbal/orm v3 will be released, adopting them should be opt-in